DTIPrep/DTIPrepRegister.pl

```diff
@@ -310,20 +369,23 @@
 #############
 
 =pod
-Register XML protocol file into mri_processing_protocol table.
-1. Check if protocol file was already registered in the database. 
-2. If protocol file already registered in the database, will return 
-the ProcessProtocolID from the database
-   If protocol file not registered yet in the database, will register
-it in the database and return the ProcessProtocolID of the file 
-registered.
-Inputs: $XMLProtocol= XML protocol file of DTIPrep to be registered
-        $data_dir   = data directory in the prod file
-        $tool       = Tool name of the protocol (a.k.a. "DTIPrep")
-Outputs:$ProtocolID = ID of the registered protocol file
-in mri_processing_protocol table that will be used to register output
-files in the files table.
+
+=head3 register_XMLProt($XMLProtocol, $data_dir, $tool)
+
+Registers XML protocol file into C<mri_processing_protocol> table. It will first
+check if protocol file was already registered in the database. If the protocol
+file is already registered in the database, it will return the
+ProcessProtocolID from the database. If the protocol file is not registered yet
+in the database, it will register it in the database and return the
+ProcessProtocolID of the registered protocol file.
+
+INPUT: XML protocol file of DTIPrep to be registered, data directory from the
```

Comment:
`INPUT` and `RETURN` formats in the entire script (see review of `register_data.pl`)